solidity/contracts/system/TBTCSystem.sol

```diff
@@ -393,14 +393,20 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     /// @notice Request a new keep opening.
     /// @param _m Minimum number of honest keep members required to sign.
     /// @param _n Number of members in the keep.
+    /// @param _stakeLockDuration Duration of stake lock in seconds.
     /// @return Address of a new keep.
-    function requestNewKeep(uint256 _m, uint256 _n, uint256 _bond)
+    function requestNewKeep(
+        uint256 _m,
+        uint256 _n,
+        uint256 _bond,
+        uint256 _stakeLockDuration
```

Comment:
I feel like we're actually making this interface too specific without actually making it specific. This isn't a stake lock duration, that's just what the Keep uses it for. It's really the max secured lifetime of the Keep, and the Keep uses this to lock the stake. Yah?

Reply:
Yes, you're 💯 correct. How about `maxSecuredLifetime`?

Reply:
I'm down.

Reply:
I updated the name in `TBTCSystem` contract context to `maxSecuredLifetime`.

But in `ECDSAKeepFactoryStub` I left it the same as is defined on the keep-ecdsa side in `IBondedECDSAKeepFactory` interface - `stakeLockDuration`. If you feel it has to be updated in this interface as well, please let me know.

Reply:
I think that makes sense on the Keep boundary, since it's very deliberately the stake being locked, not the bond.

Reply:
Eh now I'm not so sure, but I don't want to bikeshed it anymore.